manpages/lumina-info.8

```diff
@@ -0,0 +1,25 @@
+.Dd November 3, 2017
+.Dt LUMINA-INFO 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-info
+.Nd used to view information about the current desktop.
+
+.Sh SYNOPSIS
+.Nm
+
+.Sh DESCRIPTION
+.Nm
+opens a graphical interface that gives general information, license
```

Comment:
This first sentence is missing an "about": interface that ... and acknowledgements <about the installed version of Lumina?>